cmd/crio/main.go

```diff
@@ -151,13 +151,13 @@ func mergeConfig(config *server.Config, ctx *cli.Context) error {
 
 func catchShutdown(gserver *grpc.Server, sserver *server.Server, hserver *http.Server, signalled *bool) {
 	sig := make(chan os.Signal, 10)
-	signal.Notify(sig, unix.SIGINT, unix.SIGTERM)
+	signal.Notify(sig, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
I'd rather not backslide to `syscall`, which [says of itself][1]:

> Code outside the standard Go repository should be migrated to use the corresponding package in the golang.org/x/sys repository.

Instead, perhaps we can use [the cross-platform `os.Interrupt`][2] synonym instead of `SIGINT`.

I can't find a Windows analog to `SIGTERM` in the Go source; `syscall.SIGTERM` is under “[More invented values for signals][3]” there.  But some Windows docs mention `SIGTERM` (e.g. [here][4]), so maybe it does exist?  Or maybe it's just a stub?  [These docs][5] have:

> The SIGILL, SIGSEGV, and SIGTERM signals are not generated under Windows NT. They are included for ANSI compatibility. Thus you can set signal handlers for these signals with signal, and you can also explicitly generate these signals by calling raise.

I'd feel better if someone with Windows experience could speak about `signal.Notify` and how Windows daemons expect to be notified about graceful-shutdown requests.

[1]: https://golang.org/pkg/syscall/#pkg-overview
[2]: https://golang.org/pkg/os/#Signal
[3]: https://github.com/golang/go/blob/6703addeee7dfe514329289c9150df14ab2ed452/src/syscall/types_windows.go#L52
[4]: https://msdn.microsoft.com/en-us/library/ms811896.aspx#ucmgch09_topic3
[5]: https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-6.0/aa272921(v=vs.60)

Reply:
Notify is handled similar on windows. I'll switch it to synonyms of unix and windows from `golang.org/x/sys/...`